packages/react/src/ReactChildren.js

```diff
@@ -46,7 +50,7 @@ function escape(key) {
 let didWarnAboutMaps = false;
 
 const userProvidedKeyEscapeRegex = /\/+/g;
-function escapeUserProvidedKey(text) {
+function escapeUserProvidedKey(text: string): string {
   return ('' + text).replace(userProvidedKeyEscapeRegex, '$&/');
```

Comment:
I don't think this coercion is necessary because reading the key *from* an element always gives you a string (or null). But I decided not to touch this.